### DISCUSSION TITLE
K4M/Sagem S3000 reprogramming

### DISCUSSION THREAD

**Discussion by marios8543** [cedricp/ddt4all#1174] [Q&A]
Hello, this isn't an issue but a question (sorry if this is the wrong place).

Can this program be used to reprogram/calibrate the ECU after a dephaser swap, as per [Renault Technical Note 3585A](https://motorstate.com.ua/upload/product-manual/Renault-ReProg-Procedure-EN.pdf) ?
The note mentions this being done via CLIP, but you can't run that without specialized equipment.

**Comment by matzel82** [comment]
NO

**Comment by cedricp** [comment]
You can reset adaptatives learnings by manually sending "11FF" frame to the calculator, then follow the steps in the service manual. You don't need to reinject the entire software into the ECU.

**Reply by marios8543** [reply]
that's insightful, thanks a lot
